crates/sui-benchmark/src/bin/shared.rs

```diff
@@ -269,6 +268,35 @@ async fn main() {
                             .map(NextOp::Request),
                         ));
                     }
+                    NextOp::Retry(b) => {
+                        num_submitted += 1;
+                        num_error += 1;
+                        let res = qd
+                            .execute_transaction(ExecuteTransactionRequest {
+                                transaction: b.0.clone(),
+                                request_type: ExecuteTransactionRequestType::WaitForEffectsCert,
+                            })
+                            .map(move |res| {
+                                match res {
+                                    Ok(ExecuteTransactionResponse::EffectsCert(result)) => {
+                                        let (_, effects) = *result;
```

Comment:
You can put this destructuring pattern in the line right above.

Reply:
I tried, but looks like i cannot destruct the Box<> inside the EffectsCert